### DISCUSSION TITLE
Support of Hailo-8L chip

### DISCUSSION THREAD

**Discussion by xorwen** [SYSTRAN/faster-whisper#873] [Ideas]
Raspberry Foundation recently started selling "[AI Kit](https://www.raspberrypi.com/documentation/accessories/ai-kit.html)" which is a M.2 hat for connecting [Hailo-8L](https://hailo.ai/products/ai-accelerators/hailo-8l-ai-accelerator-for-ai-light-applications/) -- AI Accelerator, featuring up to 13 TOPS.

Raspberry Pi 5 can [somehat run](https://www.youtube.com/watch?v=3yLFWpKKbe8&t=1s) _tiny.en_ model on CPU, but support of Hailo-8L would allow running medium model without breaking a sweat. Any thoughts on this?

To spare you some time it look like that (unlike Coral) Hailo-8L doesn't have RAM on its own. It uses host RAM.

**Comment by trungkienbkhn** [comment]
@xorwen , unfortunately, we didn't have this device to test. Could you run a benchmark for it to compare? You can refer to the benchmark logic provided [here](https://github.com/SYSTRAN/faster-whisper/tree/master/benchmark).

**Comment by mc-hollin** [comment]
I also saw this here: [(https://community.hailo.ai/t/whisper-support-for-rpi5-hailo8l-kit/488)](https://community.hailo.ai/t/whisper-support-for-rpi5-hailo8l-kit/488/6) .
Maybe there is already communication here! ? ;-)
